app/src/main/java/com/duckduckgo/app/settings/SettingsActivity.kt

```diff
@@ -104,6 +112,10 @@ class SettingsActivity : DuckDuckGoActivity(), SettingsAutomaticallyClearWhatFra
         })
     }
 
+    private fun updateBookmarksAutoCompleteToggle(enabled: Boolean) {
+        bookmarksAutocompleteToggle.isEnabled = enabled
```

Comment:
visually, it's a little annoying that the text label for the toggle still looks enabled. However, given this is likely to be temporary, it's probably ok to leave it as is

Reply:
Agreed, the other option was to have a separate textview along with the toggle but because of it was temporary I left it like that.